src/docker-outside-of-docker/install.sh

```diff
@@ -196,14 +196,14 @@ else
         apt-get -y install --no-install-recommends moby-cli${cli_version_suffix} moby-buildx
         apt-get -y install --no-install-recommends moby-compose || echo "(*) Package moby-compose (Docker Compose v2) not available for OS ${ID} ${VERSION_CODENAME} (${architecture}). Skipping."
     else
-        apt-get -y install --no-install-recommends docker-ce-cli${cli_version_suffix}
+        apt-get -y install --no-install-recommends docker-ce-cli${cli_version_suffix} docker-buildx-plugin docker-compose-plugin
```

Comment:
> docker-compose does not work when moby: false and dockerDashCompose: v2

`docker-compose` is a v1 command, so wondering why do you think it should exist when `dockerDashCompose: v2` ?

Hence, can we remove `docker-compose-plugin`?

Reply:
First, `docker-compose-plugin` is for `docker compose` to work. It should work no matter what users set for `dockerDashCompose`.

Now, if users set `dockerDashCompose: v2`, then yes, as the option name implies, `docker-compose` commands should be transmitted to docker compose v2.

Reply:
@samruddhikhandale `docker-compose-plugin` populates `docker compose` which is always v2. (Yes this is odd). They changed their packaging I think since docker-ce-cli used to include it and most people are using moby.